File: PhP/Circle.php
<?php

include ("Point");

class Circle extends Point {
    private $x, $y, $radius;

    function  __construct($x, $y, $r) {
        $this->radius = $r;
        parent::__construct($x, $y);
    }

    function setRadius($r){
        $this->radius = $r;
    }

    function getRadius(){
        return $this->radius;
    }

    function getCircumference(){
        return (2 * 3.1416 * $this->radius);
    }

    function getArea(){
        return (3.1416 * $this->radius * $this->radius);
    }

    function printCircle(){
        echo 'Center: ';
        $this->printPoint();
        echo "Radius: " .  $this->radius .'<br />';
        echo "Circumference: " . $this->getCircumference().'<br />';
        echo "Area: "  . $this->getArea() .'<br />';
    }
}
?>

File: Python/src/Circle.py
from Point import Point

class Circle(Point):
    def __init__(self, x, y, r):
        Point.__init__(self, x, y)
        self.__radius = r
    
    def setRadius(self, r):
        self.__radius = r

    def getRadius(self):
        return self.__radius

 
    def getCircumference(self):
        return (2 * 3.1416 * self.__radius)
	  
    def getArea(self):
        return (3.1416 * self.__radius * self.__radius)

    def printCircle(self):
        print("Center: ")
        Point.printPoint(self)

        print("Radius: ", self.__radius)
        print("Circumference: ", self.getCircumference())
        print("Area: ", self.getArea())
